sw/airborne/mcu_periph/uart.h

```diff
@@ -33,15 +33,15 @@
 #include "std.h"
 
 #ifndef UART_RX_BUFFER_SIZE
-#ifdef STM32F4  //the F4 has enough memory
+#if defined STM32F4 || defined STM32F7 //the F4 and F7 have enough memory
 #define UART_RX_BUFFER_SIZE 256
 #else
 #define UART_RX_BUFFER_SIZE 128
 #endif
 #endif
 
 #ifndef UART_TX_BUFFER_SIZE
-#ifdef STM32F4  //the F4 has enough memory, and the PX4 bootloader needs more then 128
+#if defined STM32F4 || defined STM32F7 //the F4 and F7 have enough memory, and the PX4 bootloader needs more then 128
```

Comment:
I think @flixr mentioned a while ago that this can be in the board file to avoid yet another ifdef?